src/engine/facets/RulesEngineInitialFacet.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import "src/engine/facets/FacetCommonImports.sol";
+/**
+ * @title Rules Engine Initial Facet
+ * @dev This contract contains worker functions and initialization functions.
+ * @notice This contract is a critical component of the Rules Engine
+ * @author @mpetersoCode55, @ShaneDuncan602, @TJ-Everett, @VoR0220
+ */
+contract RulesEngineInitialFacet is FacetCommonImports{
```

Comment:
Hmmm...Im not sure I like the name but I'm struggling to think of an alternative.